opensuse-tomcat-image/src/main/docker/Dockerfile

```diff
@@ -17,51 +17,36 @@
 # Docker registry
 ARG DOCKER_HUB_PUBLIC=docker.io
 
-#
-# Preliminary image that downloads Tomcat and makes some modifications
-#
-FROM ${DOCKER_HUB_PUBLIC}/opensuse/leap:latest AS builder
-RUN zypper -n install gzip tar xsltproc
-WORKDIR /wd
-
-ENV TOMCAT_PARENT_DIR=/wd/root/usr/share \
-    TOMCAT_DIR_NAME=tomcat
-ENV TOMCAT_ROOT_DIR $TOMCAT_PARENT_DIR/$TOMCAT_DIR_NAME
-ENV TOMCAT_CONF_DIR $TOMCAT_ROOT_DIR/conf
-ENV TOMCAT_BIN_DIR $TOMCAT_ROOT_DIR/bin
-ENV TOMCAT_LIB_DIR $TOMCAT_ROOT_DIR/lib
-
-# Extract Tomcat and remove unwanted web applications
-COPY apache-tomcat-9.0.64.tar.gz .
-RUN mkdir -p $TOMCAT_PARENT_DIR && \
-    cd $TOMCAT_PARENT_DIR && \
-    tar xzf /wd/apache-tomcat-9.0.64.tar.gz && \
-    mv apache-tomcat-9.0.64 $TOMCAT_DIR_NAME && \
-    cd $TOMCAT_ROOT_DIR/webapps && \
-    rm -rf $TOMCAT_CONF_DIR/logging.properties docs/ examples/ host-manager/ manager/
+#OpenSUSE Tomcat Base image
+ARG JULI_IMAGE
+FROM $JULI_IMAGE AS builder
+RUN zypper -n install xsltproc
+ENV CATALINA_HOME /usr/share/tomcat
+
+ENV CATALINA_CONF_DIR $CATALINA_HOME/conf
+ENV CATALINA_BIN_DIR $CATALINA_HOME/bin
+ENV CATALINA_LIB_DIR $CATALINA_HOME/lib
+
+ENV PATH $CATALINA_BIN_DIR:$PATH
+
+WORKDIR $CATALINA_HOME
+
+RUN rm -rf $CATALINA_CONF_DIR/logging.properties
 
 # Copy new logback.xml and logback-access.xml to tomcat conf folder for logback implementation
-COPY logback*.xml $TOMCAT_CONF_DIR/
+COPY logback*.xml $CATALINA_CONF_DIR/
 
 # Copy new setenv.bat and setenv.sh to tomcat bin folder
-COPY setenv.* $TOMCAT_BIN_DIR/
+COPY setenv.* $CATALINA_BIN_DIR/
 
 # Copy a new lib folder with thirdparty jars for logback implementation to tomcat lib folder
-COPY /maven/lib/*.jar $TOMCAT_LIB_DIR/
-RUN mv $TOMCAT_LIB_DIR/caf-logging-tomcat-juli-*.jar  $TOMCAT_BIN_DIR/tomcat-juli.jar
+COPY /maven/lib/*.jar $CATALINA_LIB_DIR/
+RUN mv lib/caf-logging-tomcat-juli-*.jar bin/tomcat-juli.jar
 
 # Apply the specified transform to the server.xml file
 COPY server.xslt .
-RUN cp $TOMCAT_CONF_DIR/server.xml .
-RUN xsltproc -o $TOMCAT_CONF_DIR/server.xml server.xslt server.xml
-
-# Add in the startup scripts
-COPY startup.d/ root/startup/startup.d/
-RUN chmod +x root/startup/startup.d/*
+RUN xsltproc -o $CATALINA_CONF_DIR/server.xml server.xslt $CATALINA_CONF_DIR/server.xml
 
-#
```

Comment:
Put this comment back to make it clear that the actual image is built below.

Reply:
Done - https://github.com/CAFapi/opensuse-tomcat-image/pull/59/commits/242695b55c258bbf346f60a5fd5b28200f1513aa